SECURITY.md

```diff
@@ -110,6 +110,10 @@ You can enforce multi factor authentication (with TOTP mechanism) for Sysadmins,
 
 If eLabFTW is set up to use some external source of identity information (such as SAML auth or external auth), you can turn off the ability to log in or register using local accounts. This can be re-enabled by an administrator with direct database access, in case the IDP setup fails.
 
+### Prevent users from changing their identity
+
+Allowing users to change their email or name can be convenient, but can have implications when it comes to identity management or when interpreting changelogs. Since version 5.1.0, sysadmins can control if users are allowed to change these pieces of information. Check if it is currently allowed on your system, and consider if it is strictly required.
```

Comment:
Thanks for the update !
I would suggest this for some phrasing, feel free to judge
```suggestion
Allowing users to change their email or name can be convenient, but can have implications when it comes to identity management or when interpreting changelogs. As of [version 5.1.0](https://www.deltablot.com/posts/release-50100/#new-sysconfig-setting-to-prevent-users-from-changing-their-identity), sysadmins can restrict these identity changes. Check if it is currently allowed on your system, and assess whether it is essential for your use case.
```

Reply:
Thanks for the comments. Updated the text.